carbon-design-system/carbon-website
# docs: add carbon mid-fi kit to website

**New**

Carbon Mid-Fi kit:
- Added the kit to the "Prototyping tools" section of the "Other resources" page.

Figma page:
- Added a notification back in to notify people that the Figma kits are available to IBMers only still.

---

**Changed**
Invision Freehand kit:
- Removed written steps to download the kit.
- Added a resource card and link out to PAL guidance to download the kit.

---

**Testing**
Carbon Mid-Fi kit and Invision Freehand kit:
- Go to `Designing` --> `Other resources` --> `Prototyping tools` section
   - Make sure resource cards link out correctly.

Figma page:
- Go to `Designing` --> `Design kits` --> `Figma` tab
   - Make sure notification shows up correctly.
